docker-compose-dev.yml

```diff
@@ -0,0 +1,235 @@
+      MB_POSTGRESQL_TEST_USER: user
+    command: ["clojure", "-M:run:dev:drivers:drivers-dev:ee:ee-dev:socket"]
+    profiles:
+      - dev-app
+    networks:
+      - metabasenet
+
+# NOTE: This service isn't ready for prime time. It takes a LONG time to start
+#       up. I think there's some mounts to caches that are needed or something.
+#       Hopefully one of our node geniuses can figure it out.
+  # Just build the frontend. Mounts local dev so nothing else needed
+  fe-builder:
+    image: node:18.14
+    container_name: fe-builder
+    working_dir: /dev/metabase
+    volumes:
+      - .:/dev/metabase
+      - ~/.npm:/root/.npm
+      - ~/Library/Caches/Yarn/v6:/usr/local/share/.cache/yarn/v6
+    command:
+      - /bin/sh
+      - -c
+      - |
+        yarn install
+        yarn build-hot
+    profiles:
+      - fe-builder
+
+  # App dbs. You only need one. If running metabase containerized, use the
+  # corresponding service.
+  metabase-postgres-app-db:
+    image: postgres
+    container_name: metabase-postgres-app-db
+    restart: always
+    ports:
+      - "5432:5432"
+    environment:
+      POSTGRES_USER: mbuser
+      POSTGRES_PASSWORD: password
+      POSTGRES_DB: metabase
+    profiles:
+      - postgresappdb
+    networks:
+      - metabasenet
+
+  # If using the mysql app db, be sure to set the environment correctly for the
+  # metabase app, whether running the app containerized or locally
+  metabase-mysql-5.7-app-db:
+    image: mysql:5.7
+    container_name: metabase-mysql57-app-db
+    restart: always
+    ports:
+      - "3306:3306"
+    environment:
+      MYSQL_USER: mbuser
+      MYSQL_PASSWORD: password
+      MYSQL_ROOT_PASSWORD: password
+      MYSQL_DATABASE: metabase
+    profiles:
+      - mysql57appdb
+    networks:
+      - metabasenet
+
+  # The following *-data instances are sample warehouse dbs that can be found at
+  # https://hub.docker.com/r/metabase/qa-databases/tags.
+  # For reference, these are the env vars used for these that might be useful
+  # when configuring your app db.
+  # ENV MYSQL_DATABASE=sample
+  # ENV MYSQL_USER=metabase
+  # ENV MYSQL_PASSWORD=metasample123
+  postgres-sample-14-data:
+    image: metabase/qa-databases:postgres-sample-14
+    container_name: metabasedb_postgres_14_data
+    ports:
+      - "15432:5432"
+    restart: always
+    profiles:
+      - postgres14data
+    networks:
+      - metabasenet
+
+  mysql-sample-5.7-data:
+    image: metabase/qa-databases:mysql-sample-5.7
+    container_name: metabasedb_mysql_5_7_data
+    ports:
+      - "23306:3306"
+    restart: always
+    profiles:
+      - mysql57data
+    networks:
+      - metabasenet
+
+  mysql-sample-8-data:
+    image: metabase/qa-databases:mysql-sample-8
+    container_name: metabasedb_mysql_8_data
+    ports:
+      - "13306:3306"
+    restart: always
+    profiles:
+      - mysql8data
+    networks:
+      - metabasenet
+
+  # Plain old mongo db + UI services
+  mongo:
+    image: mongo
```

Comment:
Could we use `metabase/qa-databases:mongo-sample-...` instead?

Reply:
I changed it to use one of our mongo qa images.